sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -440,8 +443,12 @@ static rom_error_t rom_ext_boot(boot_data_t *boot_data, boot_log_t *boot_log,
     // Redundant check to ensure that the ISFB check was performed earlier in
     // the boot process.
     const manifest_ext_isfb_t *ext_isfb;
-    HARDENED_RETURN_IF_ERROR(manifest_ext_get_isfb(manifest, &ext_isfb));
-    HARDENED_CHECK_EQ(isfb_check_count, isfb_expected_count_get(ext_isfb));
+    rom_error_t error = manifest_ext_get_isfb(manifest, &ext_isfb);
```

Comment:
The ISFB extension is opt-in, not required.

Reply:
LGTM